fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/impl/registry/sync/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Contains the Registry Sync implementation.
+ */
+@ApiStatus.Internal
+package net.fabricmc.fabric.impl.registry.sync;
```

Comment:
Question, I wonder if putting a package-info.java marked with `@ApiStatus.Internal` in api-base in `net.fabricmc.fabric.impl` would cover all the modules? (not something for this PR)

Reply:
Oh, I guess it wont popagate to the sub packages (based on the next files in the PR).